src/client/Microsoft.Identity.Client/Platforms/Features/OpenTelemetry/OtelInstrumentation.cs

```diff
@@ -30,7 +31,8 @@ internal class OtelInstrumentation : IOtelInstrumentation
         private const string DurationInL1CacheHistogramName = "MsalDurationInL1CacheInUs.1B";
         private const string DurationInL2CacheHistogramName = "MsalDurationInL2Cache.1A";
         private const string DurationInHttpHistogramName = "MsalDurationInHttp.1A";
-        private const string DurationInExtensionInMsHistogram = "MsalDurationInExtensionInMs.1B";
+
+        private static Lazy<Dictionary<string, Histogram<long>>> ExtensionCounters = new(() => new Dictionary<string, Histogram<long> > ());
```

Comment:
```suggestion
        private static Lazy<Dictionary<string, Histogram<long>>> ExtensionHistogram = new(() => new Dictionary<string, Histogram<long> > ());
```

Counters and histograms are 2 different concepts in otel. Naming a histogram as counter is confusing

Reply:
reverting